### PR TITLE
[fr] translate tasks from the Managing Secrets section

### PR DESCRIPTION
**Pull request by k0rventen**
Hi,

This adds the French translation for the 3 tasks from the "Managing Secrets" section: https://kubernetes.io/docs/tasks/configmap-secret/.

There is one thing where I'm not quite sure how to translate properly. A few notes talk about the Server Side Apply concept:
`Note: The stringData field for a Secret does not work well with server-side apply.`

Translating literally  "server-side apply" would be "Application coté serveur", which is not "wrong", but IMO it's confusing, as it could be understood as an actual application hosted on a server. I've used "traitement coté serveur", which I think is closer to the reference term, but there might be a better translation.

### CODE REVIEW

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+   ```
+
+   Le résultat sera similaire à :
+
+   ```
+   secret/mysecret created
+   ```
+
+Pour vérifier que le Secret a été créé et pour décoder les données du Secret, référez-vous à la page sur la 
+[Gestion des secrets à l'aide de kubectl](/docs/tasks/configmap-secret/managing-secret-using-kubectl/#verify-the-secret).
```
Comment (sftim) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md:77]:
```suggestion
[Gestion des secrets à l'aide de kubectl](/fr/docs/tasks/configmap-secret/managing-secret-using-kubectl/#verify-the-secret).
```
?

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+Pour supprimer un Secret, utilisez `kubectl` :
+
+```shell
+kubectl delete secret db-user-pass
+```
+
+## {{% heading "whatsnext" %}}
+
+- En savoir plus sur le [concept de Secret](/docs/concepts/configuration/secret/)
```
Comment (sftim) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:151]:
```suggestion
- En savoir plus sur le [concept de Secret](/fr/docs/concepts/configuration/secret/)
```

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+- Apprendre comment [gérer les Secrets avec kubectl](/docs/tasks/configmap-secret/managing-secret-using-kubectl/)
+- Apprendre à [gérer les Secrets avec un fichier de configuration](/docs/tasks/configmap-secret/managing-secret-using-config-file/)
```
Comment (sftim) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:153]:
```suggestion
- Apprendre comment [gérer les Secrets avec kubectl](/fr/docs/tasks/configmap-secret/managing-secret-using-kubectl/)
- Apprendre à [gérer les Secrets avec un fichier de configuration](/fr/docs/tasks/configmap-secret/managing-secret-using-config-file/)
```

**Review by sftim — COMMENTED** [review]
A few nits around which page to link to

**Comment by sftim** [comment]
> traitement coté serveur

I think that works (I am not fluent in French). How about any of these:

- mise en place des modifications coté serveur
- modification coté serveur
- matérialisation coté serveur

**Comment by k0rventen** [comment]
> * des modifications

Great suggestion. I've decided to use "traitement des modifications coté serveur", which sounds pretty similar to the english counterpart. I've also added the concept name in parenthesis for completeness in https://github.com/kubernetes/website/pull/44845/commits/2935847187f17d70422383c35bce882ca9a53098

**Comment by k0rventen** [comment]
/assign @perriea 

We should be good here, unless you have an even better suggestion for my translation dilemma 😄

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+1. Convertissez le texte en base64 :
+
+   ```shell
+   echo -n 'admin' | base64
+   echo -n '1f2d1e2e67df' | base64
+   ```
+
+   {{< note >}}
+   Les valeurs JSON et YAML sérialisées des données du Secret sont encodées en base64. Les sauts de ligne ne sont pas valides à l'intérieur de ces chaînes et doivent être omis. Lors de l'utilisation de l'utilitaire `base64` sur Darwin/macOS, les utilisateurs doivent éviter d'utiliser l'option `-b` pour diviser les lignes longues. En revanche, les utilisateurs Linux *doivent* ajouter l'option `-w 0` à la commande `base64` ou alors utiliser `base64 | tr -d '\n'` si l'option `-w` n'est pas disponible.
```
Comment (perriea) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md:38]:
```suggestion
   Les valeurs JSON et YAML du Secret sont sérialisées puis encodées en base64. Les sauts de ligne ne sont pas valides à l'intérieur de ces chaînes et doivent être omis. Lors de l'utilisation de l'utilitaire `base64` sur Darwin/macOS, les utilisateurs doivent éviter d'utiliser l'option `-b` pour diviser les lignes longues. En revanche, les utilisateurs Linux *doivent* ajouter l'option `-w 0` à la commande `base64` ou alors utiliser `base64 | tr -d '\n'` si l'option `-w` n'est pas disponible.
```

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+
+Pour vérifier que le Secret a été créé et pour décoder les données du Secret, référez-vous à la page sur la 
+[Gestion des secrets à l'aide de kubectl](/fr/docs/tasks/configmap-secret/managing-secret-using-kubectl/#verify-the-secret).
+
+### Spécifier des données non encodées lors de la création d'un Secret
+
+Pour certains cas, vous pouvez
+utiliser le champ `stringData` à la place. Ce
+champ vous permet d'ajouter du texte non encodé directement dans le Secret,
+et il sera encodée pour vous lors de la création ou de la mise à jour du Secret.
```
Comment (perriea) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md:84]:
```suggestion
et il sera encodé pour vous lors de la création ou de la mise à jour du Secret.
```

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 6]
```diff
@@ -0,0 +1,248 @@
...
+
+Un exemple pratique de ce besoin pourrait être lorsque vous déployez une application
+qui utilise un Secret pour stocker un fichier de configuration, et vous voulez configurer
```
Comment (perriea) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-config-file.md:87]:
```suggestion
qui utilise un Secret pour stocker un fichier de configuration, et que vous voulez configurer
```

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+```
+{ "password": "UyFCXCpkJHpEc2I9", "username": "YWRtaW4=" }
+```
+
+```
+echo 'UyFCXCpkJHpEc2I9' | base64 --decode
+```
+
+Le résutat est similaire à :
```
Comment (perriea) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:112]:
```suggestion
Le résultat est similaire à :
```

File: content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+```
+
+Pour en savor plus, consultez
+[la gestion des secrets avec kubectl](/docs/tasks/configmap-secret/managing-secret-using-kubectl/#verify-the-secret) et la
+[Gestion déclarative des objets Kubernetes avec Kustomize](/docs/tasks/manage-kubernetes-objects/kustomization/).
+
+## Modifier un Secret {#edit-secret}
+
+1.  Dans votre fichier `kustomization.yaml`, modifiez les données, par exemple `password`.
+1.  Appliquez le dossier contenant le fichier kustomization :
```
Comment (perriea) [inline content/fr/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:125]:
```suggestion
2.  Appliquez le dossier contenant le fichier kustomization :
```

**Review by perriea — CHANGES_REQUESTED** [review]

**Pull request merged**